scripts/extensions/sams/src/ui/grid/GridItem.tsx

```diff
@@ -37,7 +38,8 @@ export class GridItem extends React.PureComponent<IProps> {
             'sd-grid-item',
             {
                 'sd-grid-item--with-click': this.props.onClick != null,
-                'sd-grid-item--selected': this.props.selected,
+                'sd-grid-item--with-click locked': this.props.doubleClickSelected,
```

Comment:
It might be more meaningful to have this property directly related to an item being locked. i.e.

`'locked: this.props.locked,`